.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
```

Comment:
What is the diff between the name to the title?

Reply:
It was automatic through Github I modified it.